File: Lib/Telerik/Examples/RichTextEditor/StructuredContent/DocumentInfo.cs
using System;

namespace RichTextEditor.StructuredContent
{
    public class DocumentInfo
    {
        #region Properties

        public string Name { get; set; }
        public string Path { get; set; }

        #endregion

        #region Constructors

        public DocumentInfo(string name, string path)
        {
            this.Name = name;
            this.Path = path;
        }

        #endregion

        #region Methods

        public override string ToString()
        {
            return this.Name;
        }

        #endregion
    }
}

File: Lib/Telerik/Examples/RichTextEditor/StructuredContent/DocumentInfo.vb
Namespace RichTextEditor.StructuredContent
    Public Class DocumentInfo
#Region "Properties"

        Public Property Name() As String
            Get
                Return m_Name
            End Get
            Set(value As String)
                m_Name = Value
            End Set
        End Property
        Private m_Name As String
        Public Property Path() As String
            Get
                Return m_Path
            End Get
            Set(value As String)
                m_Path = Value
            End Set
        End Property
        Private m_Path As String

#End Region

#Region "Constructors"

        Public Sub New(name As String, path As String)
            Me.Name = name
            Me.Path = path
        End Sub

#End Region

#Region "Methods"

        Public Overrides Function ToString() As String
            Return Me.Name
        End Function

#End Region
    End Class
End Namespace
